bears/general/FilenameBear.py

```diff
@@ -27,7 +27,9 @@ class FilenameBear(LocalBear):
 
     def run(self, filename, file,
             file_naming_convention: str=None,
-            ignore_uppercase_filenames: bool=True):
+            ignore_uppercase_filenames: bool=True,
+            prefix: str='',
+            suffix: str=''):
```

Comment:
I would call those settings `filename_prefix` and `filename_suffix` (or `file_name_prefix` / `file_name_suffix`, what you like better). There are certainly other prefixes and suffixes that mean something different in another context^^